backend/app/models/evaluation.py

```
@@ -0,0 +1,289 @@
+        foreign_key="organization.id", nullable=False, ondelete="CASCADE"
+    )
+    project_id: int = SQLField(
+        foreign_key="project.id", nullable=False, ondelete="CASCADE"
+    )
+
+    # Timestamps
+    inserted_at: datetime = SQLField(default_factory=now, nullable=False)
+    updated_at: datetime = SQLField(default_factory=now, nullable=False)
+
+    # Relationships
+    project: "Project" = Relationship(
```

Ruff (0.14.2)

Undefined name `Project`

(F821)